app/src/sharedTest/java/org/oppia/app/testing/InputInteractionViewTestActivityTest.kt

```diff
@@ -204,4 +206,72 @@ class InputInteractionViewTestActivityTest {
     onView(withId(R.id.test_fraction_input_interaction_view)).check(matches(isDisplayed()))
       .check(matches(withText("9/5")))
   }
+
+  @Test
+  fun testFractionInputInteractionView_withInputtedNegativeSymbolOtherThanAt0_numberFormatErrorIsDisplayed() {
+    ActivityScenario.launch(InputInteractionViewTestActivity::class.java)
+    onView(withId(R.id.test_fraction_input_interaction_view)).perform(typeText("55-"))
+    onView(withId(R.id.fraction_input_error)).check(matches(withText("Please enter a valid fraction (e.g., 5/3 or 1 2/3)")))
+  }
+  @Test
```

Comment:
Nit: please ensure 1 empty line in spacing between test methods.

Reply:
Done